central/apitoken/datastore/datastore_impl.go

```diff
@@ -106,3 +142,54 @@ func (b *datastoreImpl) RevokeToken(ctx context.Context, id string) (bool, error
 	}
 	return true, nil
 }
+
+func (b *datastoreImpl) Search(ctx context.Context, q *v1.Query) ([]search.Result, error) {
+	if ok, err := integrationSAC.ReadAllowed(ctx); err != nil {
+		return nil, err
+	} else if !ok {
+		return nil, nil
+	}
```

Comment:
Do we want to use `sac.VerifyAuthZOK(integrationSAC.ReadAllowed(ctx))` here instead? Looking at the references in the current datastore implementation, I don't see a reason why not to (but maybe I'm missing something?)